es6-destructuring/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>es6-destructuring</title>
+  </head>
+  <body>
+    <script>
+      const book1 = {
+  title: 'Goodnight Punpun',
+  author: 'Inio Asano',
+  libraryID: 3353
+};
+
+      const { title, author, libraryID } = book1;
+      console.log('The title of the book is,', title, 'the author is ' + author + ', and the library id is', libraryID);
```

Comment:
Use interpolation.

Reply:
i thought we shouldnt do that when console logging